modules/apps/object/object-api/src/main/java/com/liferay/object/entry/util/ObjectEntryThreadLocal.java

```diff
@@ -70,6 +73,27 @@ public static void setSkipReadOnlyObjectFieldsValidation(
 		_skipReadOnlyObjectFieldsValidation.set(skipReadOnlyValidation);
 	}
 
+	public static class ActualExpandoValue {
```

Comment:
@JhossephG I think we should follow the existing pattern in this class and not use an inner class